### PR TITLE
VBID-5071 Fix gulp watch server for CSS changes

### PR DESCRIPTION
**Pull request by RobinIsTheBird**
https://vistarmedia.atlassian.net/browse/VBID-5071

watch all the .less files

### CODE REVIEW

File: src/index.coffee [hunk 1]
```diff
@@ -28,12 +28,14 @@ isProduction = ->
...
-  static: './static/**'
-  index:  './static/index.html'
-  style:  './style/index.less'
-  test:   './test/**/*_spec.coffee'
+  dest:     './build/'
+  src:      './app/**/*.coffee'
+  static:   './static/**'
+  index:    './static/index.html'
+  stylesrc: './style/**/*.less'
+  styleapp: './app/**/*.less'
```
Comment (dmansen) [inline src/index.coffee]: Can we camelCase these? styleSrc / styleApp

**Comment by chrismcbride** [comment]
:+1:

**Comment by dmansen** [comment]
:+1:

**Comment by pbogunovich** [comment]
+2

**Pull request merged**